src/main/scala/akka/persistence/cassandra/query/EventsByPersistenceIdFetcherActor.scala

```diff
@@ -0,0 +1,124 @@
+package akka.persistence.cassandra.query
+
+import java.lang.{Long => JLong}
+import java.nio.ByteBuffer
+
+import scala.annotation.tailrec
+import scala.concurrent.{Promise, Future}
+
+import akka.actor._
+import akka.persistence.PersistentRepr
+import akka.pattern.pipe
+import akka.persistence.cassandra.query.QueryActorPublisher.ReplayDone
+import akka.serialization.{SerializationExtension, Serialization}
+import com.datastax.driver.core.utils.Bytes
+import com.datastax.driver.core.{Row, PreparedStatement, Session, ResultSet}
+
+import akka.persistence.cassandra.listenableFutureToFuture
+
+private[query] object EventsByPersistenceIdFetcherActor {
+  def props(
+      persistenceId: String, from: Long, to: Long, maxBufferSize: Int, targetPartitionSize: Int,
```

Comment:
isn't maxBufferSize and targetPartitionSize available via `CassandraReadJournalConfig`?